### ISSUE TITLE
Increased circulating dehydroepiandrosterone-sulfate concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9631]
**Preferred term label:**
Increased circulating dehydroepiandrosterone-sulfate concentration

**Synonyms**
Increased circulating DHEA-S concentration

**Definition (free text, please give PubMed ID)**
Increased circulating concentration of  dehydroepiandrosterone-sulfate, a precursor of testosterone and estrogen.

Comment: DHEA is produced in the adrenal glands and to a lesser extent by the ovaries and testes. DHEA is metabolized into DHEA-S in the adrenal glands and liver. 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
201810 
PMID: 18252794  3/3

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
